_data/ph_authors.yml

```diff
@@ -265,14 +265,15 @@
 
 
 - name: Daniel van Strien
+  orcid: 0000-0003-1684-6556
   team: false
   bio:
     en: |
-      Daniel van Strien is a Library Science student at City University, London.
```

Comment:
Extremely, extremely minor, but there's inconsistency in periods after the bios here